src/index.js

```diff
@@ -1,23 +1,23 @@
-var path = require("path");
-var findRoot = require("find-root");
-var chalk = require("chalk");
-var _ = require("lodash");
-var semver = require("semver");
+const path = require('path');
+const findRoot = require('find-root');
+const chalk = require('chalk');
+const groupBy = require('lodash.groupby');
+const semver = require('semver/semver');
 
 const defaults = {
```

Comment:
For the terra-aggregate-translations module, we opted to put src code into lib directly since we didn't compile. Do we want to follow this pattern?

Reply:
+1

Reply:
Updated